pkg/datapath/iptables/iptables.go

```diff
@@ -1565,7 +1565,6 @@ func (m *IptablesManager) installRules(ifName string) error {
 
 	if skipPodTrafficConntrack(false) {
 		podsCIDR := option.Config.GetIPv4NativeRoutingCIDR().String()
-
```

Comment:
nit: can you remove this delta from the change?

Reply:
done